pom.xml

```diff
@@ -129,7 +133,7 @@
           <plugin>
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-gpg-plugin</artifactId>
-            <version>1.5</version>
+            <version>1.6</version>
```

Comment:
it needs to be in both since it is overriding the declaration in the release profile from oss-parent.  Or do you just mean use a property?